catalog/spatial/wfs/spatial-wfs-featuretransformer-handlebars/src/main/java/org/codice/ddf/spatial/ogc/wfs/transformer/handlebars/HandlebarsWfsFeatureTransformer.java

```diff
@@ -518,34 +512,42 @@ private Serializable getValueForAttributeFormat(
   }
 
   private String getWktFromGeometry(String geometry) {
-    String wkt = getWktFromGml3(geometry);
+    String wkt = getWktFromGml(geometry, new org.geotools.gml3.GMLConfiguration());
     if (StringUtils.isNotBlank(wkt)) {
       return wkt;
     }
     LOGGER.debug("Error converting gml to wkt using gml3 configuration. Trying gml2.");
-    return getWktFromGml2(geometry);
+    return getWktFromGml(geometry, new org.geotools.gml2.GMLConfiguration());
   }
 
-  private String getWktFromGml3(String geometry) {
-    String wkt = null;
-    Gml3ToWkt gml3ToWkt = new Gml3ToWktImpl(new org.geotools.gml3.GMLConfiguration());
-    try {
-      wkt = gml3ToWkt.convert(geometry);
+  private String getWktFromGml(final String geometry, final Configuration gmlConfiguration) {
+    final Gml3ToWkt gml3ToWkt = new Gml3ToWktImpl(gmlConfiguration);
```

Comment:
❓ Will `Gml3ToWektImpl` work  if we pass in the older `org.geotools.gml2.GMLConfiguration`?

Reply:
Yes, they implement the same interface. We were already passing the GML 2 configuration object to `Gml3ToWktImpl`: see line 542 of the old file.